_rules/block-collapsible-3e12e1.md

```diff
@@ -29,7 +29,7 @@ This rule applies to any [HTML web page][].
 
 ## Expectation
 
-For each [block of repeated content][] in each test target, which is before (in the [flat tree][]) at least one node of [non-repeated content after repeated content][], all the following are true:
+For each [block of repeated content][] which is before at least one node of [non-repeated content after repeated content][] in the [flat tree][], all the following are true:
```

Comment:
I think we need to keep the "each test target" quantification to scope the "each block" one (otherwise, it could be "for each block in the entire web").
```suggestion
For each test target, and each of its [block of repeated content][] which is before at least one node of [non-repeated content after repeated content][] in the [flat tree][], all the following are true:
```

Otherwise, we might be able to play around the idea of:
```markdown
For each [applicable block](#3e12e1:applicable-block), all the following are true:
- (condition 1)
…
- (condition n)

For each test target, the <dfn id="3e12e1:applicable-block">Applicable Blocks</dfn> are the blocks of repeated content, …
```
that can split the concerns a bit and make the full easier to swallow.